app/components/access_panels/location_item_component.rb

```diff
@@ -3,16 +3,46 @@
 module AccessPanels
   class LocationItemComponent < ViewComponent::Base
     with_collection_parameter :item
-    attr_reader :item, :document, :classes
+    attr_reader :item, :document, :classes, :counter
 
-    def initialize(item:, document:, classes: nil, render_item_level_request_link: true, render_item_note: true)
+    def initialize(item:, document:, item_counter:, classes: nil, render_item_level_request_link: true, render_item_note: true)
       super
 
       @item = item
       @classes = classes
       @document = document
       @render_item_level_request_link = render_item_level_request_link
       @render_item_note = render_item_note
+      @counter = item_counter
+    end
+
+    def html_data
+      data = { barcode: item.barcode }
+      return data if counter < 5
+
+      data.merge(long_list_target: 'hideable')
+    end
+
+    def bound_with_parent?
+      item.bound_with_parent.present?
+    end
+
+    def bound_with_title
+      bound_with = item.bound_with_parent
+
+      # TODO: remove this after the index is rebuild with https://github.com/sul-dlss/searchworks_traject_indexer/pull/1379
```

Comment:
Looks like this was merged 2 weeks ago so the index should have been reindexed by now. Please remove these lines before merge.